hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+<details>
+<summary><font size=5>Table of Contents</font> </summary>
+
+- [1. Introduction](#1-Introduction)
+
+- [2. Determination of antenna matching network](#2-Determination-of-antenna-matching-network)
+  - [2.1 Why is antenna tuning important?](#11-Why-is-antenna-tuning-important?)
+  - [2.2 Tune and measure](#12-Tune-and-measure)
+
+- [3. Frequency error and HFXO Capacitor Bank(CTune) calibration](#3-Frequency-testing-and-HFXO-Capacitor-Bank(CTune)-calibration)
+  - [3.1 Why is Ctune calibration important?](#31-Why-is-Ctune-calibration-important?)
+  - [3.2 Create Railtest tool for the test](#31-Create-Railtest-tool-for-the-test)
+  - [3.3 Tune and measure](#31-Tune-and-measure)
+
+- [4. Contucted Tx fundamental power and harmonics measurement](#4-Contucted-Tx-fundamental-power-and-harmonics-measurement)
+  - [4.1 RailTest commands for Tx CW signal](#41-RailTest-commands-for-Tx-CW-signal)
+  - [4.2 The Signal Analyser setup and test example](#42-The-Signal-Analyser-setup-and-test-example)
+
+- [5. Conducted Rx sensitivity measurement](#5-Conducted-Rx-sensitivity-measurement)
+  - [5.1 The Signal Generator settings](#51-The-Signal-Generator-settings)
+  - [5.2 RailTest commands and test result](#52-RailTest-commands-and-test-result)
+  
+- [6. Radiation test environment](#6-Radiation-test-environment)
+  - [6.1 Typical radiation test chamber SAR](#51-Typical-radiation-test-chamber-SAR)
+  - [6.2 Procedures for radiated measurement](#52-Procedures-for-radiated-measurement)
+
+</details>
+
+# 1. Introduction
+As described in 4th section, the product should pass the regulation and be certificated if it will be sold in that region. Usually there is no big problem if the customer comply the design rules of Silicon Labs' reference design. But some times the customer want to customize its design to fit the final application, it needs doing some pre-tests based on customer design to pass the regulation. If some test items can not pass the regulation, then some tuning and calibration steps will be required. In the following contents, some tuning, calibration, and testing methods will be described in detail for customer reference.
```

Comment:
Change "certificated" to "certified".  Change "comply" to "comply with".

Reply:
Corrected.